desktop/analytics/bidding.js

```diff
@@ -80,8 +80,8 @@ $(document).on('click', '.my-active-bids-bid-button', function (e) {
   })
 })
 
-// Clicked "Bid" (context_type: artwork page)
-$(document).on('click', '.artwork-auction__bid-form__button', function () {
+// Clicked "Bid" (context_type: artwork page: pre-register before bidding, approval required before bidding)
+$(document).on('click', 'button.artwork-auction__bid-form__button, a.artwork-auction__bid-form__button', function () {
```

Comment:
In the registration flow, there are situations where an `a` tag is called for (href redirects) and others where the selector triggers JS. Originally I had these tracking bits broken out into separate functions as their behavior is subtly different, but after talking with analytics they said it was unnecessary and that they could derive the data elsewhere.